lib/settlement/index.js

```diff
@@ -0,0 +1,3 @@
+'use strict';
+
+module.exports = require('./settlement');
```

Comment:
This file should also export the other public units from this module (driip-settlement and null-settlement) so users dont have to "deep require" files from inside a module directory.

Reply:
Good idea, I will update it.